content/posts/2021/reana-client-python-api.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "How to run workflows using reana-client Python API"
+date: 2021-02-22T15:00:00+01:00
+tags:
+  - API
+  - client
+  - Python
+---
+
+Here you can find a step by step guide on how to submit workflows to
+REANA using the `reana-client` Python API. This is a simple example,
+for illustrative purposes, using the serial workflow engine and a
+simple [ROOT6 / RooFit analysis](https://github.com/reanahub/reana-demo-root6-roofit/) example.
+We will create a new workflow, run it and retrieve its results using
+the `reana-client` Python API.
+<!--more-->
+
+Install `reana-client`:
+```
+$ pip install reana-client==0.7.0
+```
+
+Folowing instructions can be executeed inside a Python interpreter,
+or a Python file.
+
+First we need to provide the minimum configuration:
+- Point the REANA service URL:
+```python
+# with `reana-client` it would be
+#   `export REANA_SERVER_URL=https://reana.cern.ch`
+import os
+
+os.environ['REANA_SERVER_URL'] = 'https://reana.cern.ch'
+```
+
+- Provide our REANA access token:
+```python
+# with `reana-client` it would be
+#  `export REANA_ACCESS_TOKEN=XXXXXXX`
+from getpass import getpass
+
+my_reana_token = \
+  os.getenv("REANA_ACCESS_TOKEN") or getpass('Enter your REANA token: ')
+```
+
+Now we will choose a name for the workflow and the kind of
+workflow specification we will use:
+```python
+my_workflow_name = 'root6-roofit'
+workflow_type = 'serial'
```

Comment:
`my_workflow_type` for consistency.

However, a meta comment. Since most users use Yadage, and since Yadage is actually our default workflow engine in Python API, what about switching the example to use Yadage rather than Serial?